src/ports/postgres/modules/knn/test/knn.sql_in

```diff
@@ -0,0 +1,41 @@
+m4_include(`SQLCommon.m4')
+/* -----------------------------------------------------------------------------
+ * Test knn.
+ *
+ * FIXME: Verify results
```

Comment:
You can take a look at the pivot function in the utilities folder for an example of assertion as well as the necessary license text for sql and py files.